app/channels/application_cable/channel.rb

```diff
@@ -1,4 +0,0 @@
-module ApplicationCable
-  class Channel < ActionCable::Channel::Base
-  end
-end
```

Comment:
Please stay in scope. This file was removed in another PR by another developer (that should also have stayed in scope). Potential merge conflict here...

Reply:
Noted. In this case both developers are aware of this, but obviously we see your point and will do our best to avoid things like this in the future.